src/main/java/com/cinchapi/ccl/control/order/generated/ParseException.java

```diff
@@ -0,0 +1,193 @@
+/* Generated By:JavaCC: Do not edit this line. ParseException.java Version 7.0 */
+/* JavaCCOptions:KEEP_LINE_COLUMN=true */
+package com.cinchapi.ccl.control.order.generated;
+
+/**
+ * This exception is thrown when parse errors are encountered.
+ * You can explicitly create objects of this exception type by
+ * calling the method generateParseException in the generated
+ * parser.
+ *
+ * You can modify this class to customize your error reporting
+ * mechanisms so long as you retain the public fields.
+ */
+public class ParseException extends Exception {
```

Comment:
Let's call this OrderParseException

Reply:
Hm... That one might be tough cause I realized that's a generated file. So unless we have plan on renaming it every time we generate it. I don't think I can change the generated file name